config/config.go

```
@@ -36,7 +37,8 @@ type DatabaseConfig struct {
 }
 
 type ExportConfig struct {
-	Workers int `json:"workers"`
+	Workers            int `json:"workers"`
+	ScreenshotMaxWidth int `json:"screenshotMaxWidth,omitempty"`
```

golangci-lint (2.12.2)

[error] json(snake): got 'screenshotMaxWidth' want 'screenshot_max_width'

(tagliatelle)